### DISCUSSION TITLE
Use Trained Model IN A Text to Speech Engine To Generate Voice From Text

### DISCUSSION THREAD

**Discussion by syedusama5556** [voicepaw/so-vits-svc-fork#241] [Q&A]
I wanted to ask does anyone know how can I use my trained model which I have trained on my own voice in a text-to-speech engine, to generate my own voice from text which I will provide to it If anyone have an idea how this can be done let me know if or if you have  a source code or any link regarding this that will also be helpful for me

**Comment by BillarySquintin** [comment]
I don't have a good answer, but since this program is purely voice to voice for now- a bootleg way to do this is to use an existing TTS program that can use custom voices, and then feed the output of that into you custom so-vits-svc model. I've had decent results of just using the output from a good voice from GCloud TTS and then feeding that into the Kanye voice with auto predict pitch- it sounds pretty robotic, but it works 🤷‍♂️
